src/components/Animation/AnimationCanvas.vue

```diff
@@ -249,6 +243,12 @@ export default {
     handleCancelExpired() {
       this.cancelExpired = true;
     },
+    handleLoaded() {
+      this.loaded++;
+    },
+    handleLoading() {
+      this.loading++;
+    },
```

Comment:
perhaps rename method to `incrementLoading()` / `incrementLoaded()` since it's doing a `this.loading++` / `this.loaded++`. so it's a bit more descriptive of what the function does.